scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -130,8 +133,15 @@ if [ "$silent" == "true" ]; then
 else
     kubectl ws "${kubectl_flags[@]}" "$imw"
 fi
+spaceID=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}")
 
-mbwsname=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
+if [ "$central" == "true" ]; then
+    echo "looking for synctargets in the center"
+    kubectl ws "${kubectl_flags[@]}" "$espw"
+fi
+prefixed_stname="$(kubectl get synctargets.edge --no-headers=true -o custom-columns=":metadata.name" | grep $stname)"
```

Comment:
This can go wrong in three ways. In an IS, one SyncTarget's name could be a substring of another SyncTarget's name. Such a SyncTarget name could be a substring of a kube-bind cluster namespace. And two ISes could have SyncTargets with the same name.

Better to use the ConfigMap to determine the cluster namespace name.

Reply:
I'm thinking that the script doesn't have enough information passed in to determine the name of the copied synctarget. I.e. the only parameter `synctarget_name` is not sufficient, the caller of this script should also pass in the kcp workspace ID. What do you think?

Reply:
IIRC, this script is invoked when the MBS is the current workspace, so the situation is not ambiguous. Perhaps I am remembering wrong. I see that this script is also told the IS, this also removes ambiguity.

Reply:
Looking at the `kcp.io/cluster` annotation on any object in a workspace will tell you the `logicalcluster.Name` of that workspace.  Every workspace has a Namespace object named `default`.

Reply:
Thanks for the suggestions. Addressed this in #1334.

Reply:
done